features/order/managing_orders/seeing_order_item_included_taxes.feature

```diff
@@ -5,24 +5,24 @@ Feature: Seeing included in price taxes of order items
     I want to see included in price taxes value of a specific order
 
     Background:
-        Given the store operates on a single channel in "France"
-        And there is a zone "EU" containing all members of the European Union
-        And default tax zone is "EU"
-        And the store has included in price "Guns tax" tax rate of 10% for "Guns" within "EU" zone
-        And the store has a product "Winchester M1866" priced at "€220.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And default tax zone is "NA"
+        And the store has included in price "Guns tax" tax rate of 10% for "Guns" within "NA" zone
+        And the store has a product "Winchester M1866" priced at "$220.00"
         And it belongs to "Guns" tax category
-        And the store ships everything for free within "EU" zone
+        And the store ships everything for free within "NA" zone
```

Comment:
Do we need `NA` zone at all? As default there is a `US` zone created so we can use it for this example.